### DISCUSSION TITLE
Latest Commits Broke Font Config

### DISCUSSION THREAD

**Discussion by chxp82q** [wezterm/wezterm#697] [Q&A] [answered]
Hello,

I believe the latest commits broke my font configuration.

[Here](http://0x0.st/-AoQ.txt) is my config file with the font options. Wezterm does not load the font at all and gives me this error:

![image](https://user-images.githubusercontent.com/33443763/114872188-ee21e080-9dae-11eb-9050-79448726c791.png)

It worked before the latest commits, so I'm wondering if I missed a change to the options?

**Comment by wez** [comment]
Please run: `WEZTERM_LOG=wezterm_font=trace,info wezterm ls-fonts` and share the output!

**Reply by chxp82q** [reply]
@wez Thanks for the reply!
[stdout](http://0x0.st/-AXu.txt)
[stderr](http://0x0.st/-AXQ.txt)

The font I am trying to use is Sarasa Mono K, and it was working fine before the latest commits.

**Reply by wez** [reply]
What does `fc-match -v "Sarasa Mono K"` output?

**Reply by chxp82q** [reply]
@wez Here it is [output](http://0x0.st/-AX2.txt)

**Reply by wez** [reply]
I'm about to get tied up with work for the day; in the meantime try using the postscriptname for that font: `Sarasa-Mono-K-Regular`

**Reply by chxp82q** [reply]
@wez Any updates on the issue? Also, how would I use the postscriptname? Also, this seems to be a problem specifically for this font.

**Reply by wez** [reply]
you can use the postscript name in the place of the family name: eg: `wezterm.font("Sarasa-Mono-K-Regular")`.  This is (perhaps not very clearly) explained in https://wezfurlong.org/wezterm/config/lua/wezterm/font.html

Would you mind also running: `fc-list 'Sarasa Mono K'` and sharing its output?

The issue appears to be that the `fc-list` equivalent query returns no matches for that font when run in code; I'd like to see what that command outputs to compare.

**Reply by chxp82q** [reply]
@wez 
![image](https://user-images.githubusercontent.com/33443763/115068586-4c31ef00-9ea7-11eb-90c1-051de83ea2ce.png)

Again, thanks for looking into the issue!

Wezterm still doesn't recognize the font after using the postscript name `wezterm.font("Sarasa-Mono-K-Regular")`

**Reply by chxp82q** [reply]
so for now i have rolled back to wezterm stable

**Reply by chxp82q** [reply]
@wez any news? Thanks!

**Comment by chxp82q** [comment, marked as answer]
Solved in #726, thanks!
